src/test/run-make/issue-26995/Makefile

```diff
@@ -0,0 +1,6 @@
+-include ../tools.mk
+
+all:
+	printf "#[path=\"%s/%s\"] pub mod baz;\n" `pwd` "bar/baz.rs" > foo.rs
```

Comment:
Hm ok, I just try to avoid run-make like the plague as makefiles are basically impossible to get right on both windows and unix...